.github/CODEOWNERS

```
@@ -66,6 +66,7 @@
 /jfrog_platform/                         @jfrog/partner-engineering
 /ilert/                                  @yacut support@ilert.com
 /insightfinder/                          @dearmore @erinlmcmahon support@insightfinder.com
+/isdown/				 @ntomas support@isdown.app
```

Our validation here is a bit nitpicky about tabs vs spaces, make sure it's only spaces.

```suggestion
/isdown/                                 @ntomas support@isdown.app
```